packages/api-gateway-service/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "api-gateway",
+  "description": "API Gateway for One Platform",
+  "opservice": {
+    "className": "ApiGateway",
+    "serviceName": "api-gateway"
+  },
+  "version": "0.0.1",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:1-Platform/one-platform",
+    "directory": "packages/api-gateway-service"
+  },
+  "publishConfig": {
+    "access": "private"
+  },
+  "private": true,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build:dev": "webpack --config webpack.dev.js",
+    "start:dev": "webpack --watch --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack --watch --config webpack.prod.js"
+  },
+  "author": {
+    "name": "Rigin Oommen",
+    "email": "riginoommen@gmail.com"
```

Comment:
Use your red hat email here.

Reply:
As per todays discussion i think we can close this